gossip/monitor/monitor.go

```diff
@@ -194,21 +203,18 @@ func (m Monitor) sendAlert(msg string) {
 	}
 }
 
-func (m Monitor) executeTask(task QueryTask) {
-	log.Debug("Executing task: %+v", task)
-	resp, err := m.client.Incremental(task.Start, task.End)
+func (q QueryTask) Do() {
+	log.Debug("Executing task: %+v", q)
+	resp, err := q.client.Incremental(q.Start, q.End)
 	if err != nil {
-		// TODO: retry
 		metrics.QedMonitorGetIncrementalProofErrTotal.Inc()
-		log.Infof("Unable to get incremental proof from QED server: %s", err.Error())
+		log.Infof("Unable to verify incremental proof from %d to %d", q.Start, q.End)
```

Comment:
This error message doesn't tell the true error source. The former was correct.

Reply:
Fixed.